config/rootfs/chromiumos/scripts/build_board.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -e
+BOARD=$1
+DATA_DIR=$(pwd)
+BRANCH="release-R99-14469.B"
+
+# Building SDK itself
+if [ ! -d chromiumos-sdk ]; then
+  sudo mkdir chromiumos-sdk
+  sudo chown user chromiumos-sdk
+  cd chromiumos-sdk
+  git config --global user.email "bot@kernelci.org"
+  git config --global user.name "KernelCI Bot"
+  git config --global color.ui false
+  repo init --repo-url https://chromium.googlesource.com/external/repo --manifest-url https://chromium.googlesource.com/chromiumos/manifest --manifest-name default.xml --manifest-branch ${BRANCH}
```

Comment:
How about `--depth=1` for shallow clones? Do we need full clones?

Reply:
Done! (and tested)

Reply:
I don't see it in the diff though

Reply:
I will recheck and retest that.